analyzer/src/main/kotlin/MavenSupport.kt

```diff
@@ -329,7 +348,7 @@ class MavenSupport(localRepositoryManagerConverter: (LocalRepositoryManager) ->
             } catch (e: ProjectBuildingException) {
                 e.showStackTrace()
 
-                val failedProject = e.results.find { projectBuildingResult ->
+                val failedProject = e.results?.find { projectBuildingResult ->
```

Comment:
Should this, strictly speaking, be a separate commit as it seems to be a fixup for 5c7d98e2c9e25713dbef3eb1e1461c01eccf1f4f?

Reply:
Done.